src/Media.Plugin.Abstractions/IMedia.cs

```diff
@@ -62,5 +62,10 @@ public interface IMedia
         /// <returns>Media file of new video or null if canceled</returns>
         Task<MediaFile> TakeVideoAsync(StoreVideoOptions options);
 
-    }
+		/// <summary>
+		/// Capture a screenshot of the current view
+		/// </summary>
+		/// <returns>Screenshot image</returns>
+		Task<byte[]> CaptureScreenshotAsync();
```

Comment:
Instead of being a byte[] can we make this a MediaFile. We should save it to disk like other APIs